lib/Doctrine/Common/Persistence/Event/OnClearEventArgs.php

```diff
@@ -8,24 +8,17 @@
  * Provides event arguments for the onClear event.
  *
  * @link   www.doctrine-project.org
- * @since  2.2
- * @author Roman Borschel <roman@code-factory.de>
- * @author Benjamin Eberlei <kontakt@beberlei.de>
  */
 class OnClearEventArgs extends EventArgs
 {
-    /**
-     * @var \Doctrine\Common\Persistence\ObjectManager
-     */
+    /** @var ObjectManager */
     private $objectManager;
 
-    /**
-     * @var string|null
-     */
+    /** @var string|null */
     private $entityClass;
 
     /**
-     * Constructor.
+     *
      *
      * @param ObjectManager $objectManager The object manager.
```

Comment:
Unfortunately we currently have no sniff for phpDoc spacing so some fixers leave superfluous empty lines:
- empty line(s) after `/**`
- empty line(s) before `*/
- multiple empty lines instead of single empty line

Can you grep through these manually? It's slightly annoying, but doable with three regexes.